MapleServer2/PacketHandlers/Game/PartyHandler.cs

```diff
@@ -316,6 +320,11 @@ private static void HandleVoteKick(GameSession session, PacketReader packet)
             //TODO: Keep a counter of vote kicks for a player?
         }
 
+        public static void HandleSummonParty(GameSession session, PacketReader packet)
+        {
+            System.Console.WriteLine("implement summon party");
```

Comment:
This should probably just be a TODO comment

Reply:
changed.